packages/snaps-execution-environments/src/common/BaseSnapExecutor.ts

```diff
@@ -46,6 +46,7 @@ import {
 import {
```

Comment:
This PR should get latest version of `utils` when published after this PR is merged: https://github.com/MetaMask/utils/pull/103